diff --git a/js/overrides/tabledrag.js b/js/overrides/tabledrag.js
index 2462c081f..16dd901f7 100644
--- a/js/overrides/tabledrag.js
+++ b/js/overrides/tabledrag.js
@@ -253,7 +253,8 @@
         this.toggleColumns();
       }.bind(this),
     );
-    if ($table.parents('.gin-table-scroll-wrapper')) {
+
+    if ($table.parents('.gin-table-scroll-wrapper').length > 0) {
       $table.parents('.gin-table-scroll-wrapper').before($toggleWeightWrapper);
     }
     else {
